src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVService/CVObjectDetectionRequest.cs

```diff
@@ -35,8 +35,8 @@ public class CVObjectDetectionRequest
         public int index;
 
         public CVObjectDetectionRequest(CVImageBinaryData screenshot,
-                                        string? textQuery,
-                                        CVImageBinaryData? queryImage,
+                                        [CanBeNull] string textQuery,
+                                        [CanBeNull] CVImageBinaryData queryImage,
```

Comment:
this should be imageQuery to be consistent with the model and other naming.

Really, we don't need the word 'query' on these at all